src/patternfly/components/Pagination/pagination.scss

```diff
@@ -255,6 +255,19 @@ $pf-c-pagination--breakpoint-map: build-breakpoint-map();
     }
   }
 
+  // hide number input up/down arrow buttons
+  input[type="number"] {
```

Comment:
Can you test adding these styles under the `.pf-c-form-control` selector below? That selector should target this element without the need to define `input[...]`